Src/PerformanceCollector/Shared/Implementation/StandardPerformanceCollector/StandardPerformanceCollector.cs

```diff
@@ -294,9 +294,9 @@ private void RegisterPerformanceCounter(string originalString, string reportAs,
                         usesInstanceNamePlaceholder,
```

Comment:
Indeed, certain counters we want to be retrying for every 5 minutes, so we add them to the list regardless.